docs/threads-in-pruntime.md

```diff
@@ -0,0 +1,18 @@
+# Threads in pruntime
+
+| Threads | Number of threads   | Description|
+| ------- | --------------- | ------------- |
+| main thread | 1  | the entry of the program  |
+| benchmark | --cores  | threads to run the benchmark |
```

Comment:
Are they the same cores to run ink queries?

Reply:
Yes